src/utils/rules-proxy.js

```diff
@@ -33,11 +33,11 @@ const getLanguage = function (headers) {
   return 'en'
 }
 
-const getCountry = function () {
-  return 'us'
+const getCountry = function (geoCountry) {
+  return geoCountry || 'us'
 }
```

Comment:
Discussed w/ @kitop on Slack. Not sure why this fallback to `us` exists but we decided to preserve it for backwards compatibility.